openpgp.d.ts

```diff
@@ -521,14 +521,14 @@ export class SignaturePacket extends BasePacket {
   public issuerFingerprint: null | Uint8Array;
   public preferredAEADAlgorithms: enums.aead[] | null;
   public revoked: null | boolean;
-  public rawNotations: RawNotation[];
+  public rawNotations: Notation[];
   public sign(key: AnySecretKeyPacket, data: Uint8Array, date?: Date, detached?: boolean): Promise<void>;
   public verify(key: AnyKeyPacket, signatureType: enums.signature, data: Uint8Array | object, date?: Date, detached?: boolean, config?: Config): Promise<void>; // throws on error
   public isExpired(date?: Date): boolean;
   public getExpirationTime(): Date | typeof Infinity;
 }
 
-export interface RawNotation {
+export interface Notation {
```

Comment:
I would rather prefer to get rid of `notations` in v6 and rename `rawNotations` to `notations`, as the latter is the more general interface, and `notations` only supports human-readable notations (but `rawNotations` was added instead of changing `notations`, to keep backwards compatibility)

Reply:
But we could also keep the name the same for now and rename them together, yeah